apps/rush-lib/src/logic/InstallManager.ts

```diff
@@ -737,18 +737,29 @@ export class InstallManager {
 
           // create the new tarball
           tar.create({
-            gzip: true,
+            gzip: false,
             file: tarballFile,
             cwd: tempProjectFolder,
             portable: true,
             noMtime: true,
             noPax: true,
             sync: true,
-            prefix: npmPackageFolder
+            prefix: npmPackageFolder,
+            filter: (path: string, stat: tar.FileStat): boolean => {
+              if (this._rushConfiguration.experimentsConfiguration.configuration
+                .normalizeChmodFieldInTarHeader) {
```

Comment:
Should `normalizeChmodFieldInTarHeader` default to `true` instead of `false`?  It seems like an important fix with minimal fix.